tests/tests/messages.rs

```diff
@@ -70,7 +70,10 @@ async fn can_submit_genesis_message() {
     assert!(
         matches!(status, TransactionStatus::Success { .. }),
         "expected success, received {status:?}",
-    )
+    );
+    loop {
+        tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
+    }
```

Comment:
```suggestion
```

This should be removed. I presume it was added to repro the issue locally?

Reply:
ah yeh whoops